packages/typespec-ts/src/modular/buildProjectFiles.ts

```diff
@@ -265,6 +265,12 @@ export function emitPackage(
       "./dist-esm/test/public/utils/env.js":
         "./dist-esm/test/public/utils/env.browser.js"
     };
+
+    packageInfo["mocha"] = {
```

Comment:
Is this config required for SDK repo also?

Reply:
openai has similar config in a mocha config file

Reply:
We could switch to a separate config file in another PR if desired.

Reply:
No strong preferences here and I think it's fine if the config is quite simple.